src/TextInput/TextInput.tsx

```diff
@@ -120,16 +121,15 @@ interface IInput {
   error: boolean
   disabled?: boolean
   value?: string
-}
-
-interface IInputOutline extends IInput {
   outlined?: boolean
 }
 
-const Content = styled.div<IInputOutline>`
+const Content = styled.div<IInput>`
   border-bottom: 1px solid;
   border-color: ${({ error }) =>
     theme.colors[`${error ? 'error' : 'outline'}`]};
+  background-color: ${({ outlined }) =>
+    outlined ? 'transparent' : theme.colors['white']};
```

Comment:
Since I removed the background on the input itself, I put it back here
When it is the outline, I felt that not having one made sense

In most case, it wouldn't make a difference since we have mostly white background tho